src/app/shared/pipes/date-timezone.pipe.ts

```
@@ -0,0 +1,16 @@
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import timezone from 'dayjs/plugin/timezone';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+
+@Pipe({
+  name: 'dateTimezone',
+})
+export class DateTimezonePipe implements PipeTransform {
+  transform(value: string, timezone: string, format: string = 'MMM DD, YYYY'): string {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)